geometry_msgs/Pose[] poses
int32[] class_ids
sensor_msgs/PointCloud[] point_clouds